src/wasm-lib/grackle/src/native_functions/sketch/helpers.rs

```diff
@@ -139,7 +139,7 @@ pub fn sequence_binding(
     }
 }
 
-/// Extract a 2D point from an argument to a Cabble.
+/// Extract a 2D point from an argument to a Cabble. <- What's a Cabble?
```

Comment:
```suggestion
/// Extract a 2D point from an argument to a KCL function
```
Wow, what a weird typo.